### PR TITLE
Star System

### PR DESCRIPTION
**Pull request by LilCSharp**
### What is a quick description of the change?

Added system to add stars

### Screenshot/Screengrab after this change?

image

### Is this fixing an issue?

Adding the ability to add stars

### Are there more details that are relevant?

none

### CODE REVIEW

**Comment by aliciasimon** [comment]
LGTM

File: src/main/java/com/google/sps/servlets/GetProductDetails.java [hunk 1]
```diff
@@ -59,6 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     long id = Long.parseLong(request.getParameter("id"));
 
     long cost;
+    double rating = 0;
```
Comment (wu-haotian) [inline src/main/java/com/google/sps/servlets/GetProductDetails.java:64]: I think double by default have value 0.0, thus you can remove the "=0".

File: src/main/java/com/google/sps/servlets/SendRating.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import com.google.gson.Gson;
+
+@WebServlet("/sendRating")
+public class SendRating extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) 
+    throws IOException {
+    
+
```
Comment (wu-haotian) [inline src/main/java/com/google/sps/servlets/SendRating.java:29]: nit: remove an extra line

**Review by wu-haotian — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/GetProductDetails.java (hunk 1 above)
Comment (LilCSharp) [reply src/main/java/com/google/sps/servlets/GetProductDetails.java:64]: It should, but the compiler through an uninitialized variable warning, so I put it there to be safe.

**Pull request merged**